src/app/series/directives/series-templates.component.html

```diff
@@ -0,0 +1,55 @@
+<form *ngIf="series">
+  <prx-fancy-field label="Audio Templates">
+    <div class="fancy-hint">
+      When you add episodes, you may want to have different versions of the audio (e.g., clean v. explicit).
+      This page lets you define templates for the versions each episode in this series should have, including
+      specific segment requirements (note: don't include ads here). Every episode using these templates will
+      be checked against them for validity.
+    </div>
+  </prx-fancy-field>
+
+  <ng-container *ngFor="let v of series.versionTemplates">
+    <div *ngIf="!v.isDestroy" class="version">
+      <header>
+        <strong>{{v?.label}}</strong>
+        <button type="button" class="btn-icon icon-cancel grey-dove" (click)="confirmRemoveVersion(v)"></button>
```

Comment:
btn-icon should have `aria-label` for accessibility. (I put it in the styleguide, so this is basically a rule now. 😃 )